README.md

```diff
@@ -15,3 +15,4 @@ to explore ArgoCD and GitOps!
 | [helm-dependency](helm-dependency/) | Demonstrates how to customize an OTS (off-the-shelf) helm chart from an upstream repo |
 | [blue-green-deploy](blue-green-deploy/) | Demonstrates an Argo CD Sync hook which performs a blue/green deployment |
 | [sock-shop](sock-shop/) | A microservices demo application (https://microservices-demo.github.io) |
+| [kustomized-helm](kustomized-helm/) | Demonstrates an application comprised of a helm chart and customized using `kustomize` |
```

Comment:
` kustomized-helm` -->  `customized-templating-tool`? confusing? makes me think of kustomize, or helm

Reply:
yeah I would also suggest framing this as a way to use any user specified template tool, it just happens to use kustomize in this example

Reply:
Agree. Added one more example which uses [Kasane](https://github.com/google/kasane) and moved both into `/plugins` directory. @alexec , @limnick do you think it looks better now?

Reply:
looks good to me! Thanks for the changes!

I haven't tried running through the steps myself to confirm they work as written.